Gemfile

```diff
@@ -2,5 +2,5 @@ source 'https://rubygems.org'
 gemspec
 
 group :development do
-  gem 'smart_proxy', :git => "https://github.com/theforeman/smart-proxy"
+  gem 'smart_proxy', :git => 'https://github.com/theforeman/smart-proxy', :branch => 'develop'
```

Comment:
This seems to be the standard for group development.
see:
https://github.com/theforeman/smart_proxy_remote_execution_ssh/blob/master/Gemfile#L12
https://github.com/theforeman/smart_proxy_discovery/blob/develop/Gemfile#L8
https://github.com/theforeman/smart_proxy_openscap/blob/master/Gemfile#L7
https://github.com/theforeman/smart_proxy_dns_plugin_template/blob/master/Gemfile#L5
https://github.com/theforeman/smart_proxy_chef/blob/master/Gemfile#L7
to name a few